src/internet_identity/src/main.rs

```diff
@@ -541,6 +544,31 @@ mod v2_api {
     }
 }
 
+/// API for the attribute sharing mvp
+mod attribute_sharing_mvp {
+    use super::*;
+
+    #[update]
+    #[candid_method]
+    fn prepare_principal_link(
```

Comment:
Please document briefly these functions, maybe explaining also what is a "principal link".